README.md

```diff
@@ -51,267 +51,72 @@ To run all tests whose name begins with `test_async_scheduler`, you can use the
 pytest -k test_async_scheduler
 ```
 
-## How to enable tuning and tuning script conventions
 
-This section describes how to enable tuning an endpoint script. In particular, we describe:
+## Getting started
```

Comment:
List above syne-tune[X]: Add 'kde' and 'blackbox-repository' as tags

Reply:
Maybe also say that 'raytune' installs all Ray dependencies, which is quite a big chunk.

Reply:
Comment on 'pytest -k ...": Is this needed?

Reply:
The line for pytest was added by Iaro IIRC, I added kde but blackbox-repo as we dont mention it yet too much in our doc (users can get it by just doing extra).